### PR TITLE
Adding Notifcations

### PR DESCRIPTION
**Pull request by aastha01chauhan**
-This is to send a notification to the user 2 days before the expected date
-Requires some fixing

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ android {
 
     defaultConfig {
         applicationId "org.openlake.sampoorna"
-        minSdk 21
+        minSdk 26
```
Comment (akshaaatt) [inline app/build.gradle:15]: We don't want to do this in any case. Look for alternatives if this is a necessity in the current implementations.

File: app/src/main/java/org/openlake/sampoorna/presentation/features/periodtracker/Notifications01.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+
+const val notificationID = 1
+const val channelID = "channel1"
+const val titleExtra = "titleExtra"
+const val messageExtra = "messageExtra"
+
+class Notifications01 : BroadcastReceiver()
```
Comment (akshaaatt) [inline app/src/main/java/org/openlake/sampoorna/presentation/features/periodtracker/Notifications01.kt:15]: The class name here can be better, based on the content of the usage.

File: app/src/main/res/layout/fragment_tracking.xml [hunk 3]
```diff
@@ -30,25 +33,26 @@
         android:text="@string/Countdown"
         android:textColor="@color/Black_Nero1"
         android:textColorHint="@color/Black_Mitsubishi_Black_Matt"
-        android:textSize="@dimen/_24ssp"
+        android:textSize="34sp"
```
Comment (akshaaatt) [inline app/src/main/res/layout/fragment_tracking.xml:36]: We should look to use the @dimen/ implementation here

**Review by akshaaatt — CHANGES_REQUESTED** [review]
Great job! The work looks promising to me here, though have requested some changes. Let me know if you need any help.

**Comment by KShivendu** [comment]
@aastha01chauhan Please do the requested changes. We'll be in a position to merge your code then.

**Pull request closed without merging**